charts/acp/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: acp
 description: Cloudentity Authorization Control Plane
 type: application
-version: 0.16.3
+version: 0.16.4
```

Comment:
We will release `0.17.0`, can we wait with this PR?

Reply:
CAn't we release 0.16.4 and 0.17.0 ?

Reply:
0.17.0 will be released today together with ACP 1.17.